students/alxsk/session10/virtualenv/iterator_1.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+"""
+Simple iterator examples
+"""
+
+
+class IterateMe_1:
```

Comment:
can you change this to take in a start, stop, and step? Also can you see how to make this run like the range function?